src/app/atomic/atm.button/button.component.style.ts

```diff
@@ -0,0 +1,27 @@
+// components/Button/styles.ts
+import { tv } from "tailwind-variants";
+
+export const buttonVariants = tv({
+  base: "h-10 w-47 px-4 py-2 rounded-sm font-medium font-semibold",
```

Comment:
embora assim funcione, é melhor se colocarmos as medidas, ex:
```suggestion
  base: "h-[10px] w-[47px] px-[4px] py-[2px] rounded-sm font-medium font-semibold",
```
dá mais clareza pra quem lê o código

também vale a pena dar uma olhada [aqui](https://m2.material.io/components/buttons), é um guideline da google pra desenhar botões. entre as sugestões, uma que buscamos adotar é de usar [a proporção 1height-2width, ou seja, 24px de altura por 48px de largura](https://m2.material.io/components/buttons#specs)

se possível, também é bom que os tamanhos estejam dentro do seu [theme do tailwind](https://v1.tailwindcss.com/docs/customizing-spacing), assim conservamos a consistência dos espaçamentos ao longo do app!

Reply:
Entendi! Já apliquei o tema e vou dar uma olhada na guideline do google. As medidas para esse botão foram pegas do figma